plugin/trino-iceberg/pom.xml

```diff
@@ -196,7 +196,7 @@
         <dependency>
             <groupId>org.apache.datasketches</groupId>
             <artifactId>datasketches-java</artifactId>
-            <version>5.0.2</version>
+            <version>6.0.0</version>
```

Comment:
Is this new major version stable?

Reply:
it seems that version was bumped because new APIs were added:

```
New: quantiles T-Digest sketch
New: BloomFilter
New: Exact and Bounded Sampling Proportional to Size (EB-PPS) sketch
```